docs/source/refs.bib

```diff
@@ -78,3 +78,11 @@ @inproceedings{kissinger2019PyZXLargeScale
   booktitle = {Quantum {{Physics}} and {{Logic}}},
   url = {https://arxiv.org/pdf/1904.04735.pdf}
 }
+
+@article{peham2022equivalenceCheckingZXCalculus,
+    title = {Equivalence Checking of Quantum Circuits with the ZX-Calculus},
```

Comment:
```suggestion
    title = {Equivalence Checking of Quantum Circuits with the {ZX}-Calculus},
```
Could you also maybe fix the `{DD}` in https://github.com/cda-tum/qcec/blob/71c78d3bfc11852b08b890a4a5f2e1024165437c/docs/source/refs.bib#L4?
And the `{A}` here https://github.com/cda-tum/qcec/blob/71c78d3bfc11852b08b890a4a5f2e1024165437c/docs/source/refs.bib#L67?